### ISSUE TITLE
Unable to create a library

### ISSUE THREAD

**Issue by sagarjoshi3183** [Librarika/Issues#165]
I logged into my account and tried to create a library but every time I do so, I get the error 

Last Action: The library could not be saved. Please, try again.

Everything filled out in the library creating form seems to be okay and there's no error/highlighting shown in the form. Just a message at the top that the library could not be saved. Please, try again.

**Comment by rayhan** [comment]
Hi, you need a fresh email address to create a new library, your email address has already been used. Please logout from the librarika and try using a fresh email address.

Thanks

**Comment by sagarjoshi3183** [comment]
Hi Rayhan,

I tried using a different email id also but it didn't work. Moreover, the email id that I was using was used to create a library that was deleted by you upon my request in a previous thread (#159). You had removed the email id from the records too at that time.

Thanks

**Comment by rayhan** [comment]
Ok, I am checking this, reopened this thread.

**Comment by rayhan** [comment]
Please try now, hopefully you can create a new library using your email address. It is very unlikely that it will still create a problem, but please let me know.

**Comment by sagarjoshi3183** [comment]
Hi Rayhan,

It has deleted my email id and the previous library from its records. I have been able to register for a new library with the same email ID and am now waiting for the activation mail. However, given that the past records have been removed completely (I tried forgot password but it did not recognize my email), I hope there should not be any problem. I'll post the updates as soon as I am able to activate the library and make it functional. 

Thanks for the help. It surely is a great platform to work on. :)
